app/services/katello/pulp3/migration.rb

```diff
@@ -35,11 +35,15 @@ def migration_plan_api
       end
 
       def pulp2_content_api
-        Pulp2to3MigrationClient::Pulp2contentApi.new(api_client)
+        Pulp2to3MigrationClient::Pulp2ContentApi.new(api_client)
+      rescue NameError
+        Pulp2to3MigrationClient::Pulp2contentApi.new(api_client) #backwards compatible
       end
 
       def pulp2_repositories_api
-        Pulp2to3MigrationClient::Pulp2repositoriesApi.new(api_client)
+        Pulp2to3MigrationClient::Pulp2RepositoriesApi.new(api_client)
+      rescue NameError
+        Pulp2to3MigrationClient::Pulp2repositoriesApi.new(api_client) #backwards compatible
```

Comment:
i left the old constants here so we can cherry pick this and have more flexibility with packaging.  It will just work everywhere